Content.Shared/CCVar/CCVars.Salvage.cs

```diff
@@ -8,7 +8,7 @@ public sealed partial class CCVars
     ///     Duration for missions
     /// </summary>
     public static readonly CVarDef<float>
-        SalvageExpeditionDuration = CVarDef.Create("salvage.expedition_duration", 900f, CVar.REPLICATED); // Frontier: This is not used, look for SalvageTimeMod
+        SalvageExpeditionDuration = CVarDef.Create("salvage.expedition_duration", 1200f, CVar.REPLICATED); // Frontier: This is not used, look for SalvageTimeMod // LoP Edit, 900f => 1200f
```

Comment:
```suggestion
        SalvageExpeditionDuration = CVarDef.Create("salvage.expedition_duration", 900f, CVar.REPLICATED); // Frontier: This is not used, look for SalvageTimeMod
```
Вообще как-бы это меняется в конфигах сервака а не в CCVars...